### ISSUE TITLE
the sass file could use some variable goodness

### ISSUE THREAD

**Issue by chrisdrackett** [nathansmith/formalize#49]
being sass and all

**Comment by nathansmith** [comment]
What aspects would you like to see variable-ized?

**Comment by chrisdrackett** [comment]
I just saw a lot of repeated colors and sizes. Seems if you are going to use sass, might be worth creating maybe a mixin so users can customize the colors/sizes  by changing a couple of clear variables

**Comment by nathansmith** [comment]
Yeah, not disagreeing, was just wondering if you had specifics you'd like to see variable-ized. Cool, cool.

**Comment by chrisdrackett** [comment]
yeah, sorry ;)

I actually wrote this issue and was planning on doing it myself. Then I decided that formalize has a bit too many default styles for my use, and I've started writing my own "reset" that in theory will be theme-able.

**Comment by nathansmith** [comment]
This might be of interest...

https://gist.github.com/1599212

It removes _all_ styling of form elements, so that they can be built back up again.

**Comment by chrisdrackett** [comment]
nice, thanks :)

**Comment by chrisvanpatten** [comment]
I'd love to see Formalize variablized too. It would make it easy to deploy customized versions.

**Comment by henrijs** [comment]
For me main issue is padding of form elements. This line in particular is evil "height: 1.8em;" If I add 3 pixels of padding above and below, by how much should I increase height? (Rethorical question). Sass could calculate that probably.

Also font-family could be as one mixin.

**Comment by nathansmith** [comment]
@henrijs — Sure, that could be automated (made dependent on a variable), but not reliably calculated. I mean, it's going to be contingent on whatever visual effect you're trying to achieve. I settled on 1.8em because that makes textual inputs the same height as a `<select>` drop-down.
